src/lib/selectors/local/main.py

```diff
@@ -0,0 +1,42 @@
+from lib.common.selector import Selector
+import pandas as pd
+import os
+from shutil import copyfile
+
+
+class LocalSelector(Selector):
+    def index(self, config):
```

Comment:
Maybe a brief class doc here about the limitations of this local selector as we discussed-- i.e. how it assumes separate media files should go into separate elements, how it derives the element 'id', etc.

On the note of ID derivation, selectors should really be responsible for ensuring that each element they select has a unique ID. So we may need some more thorough checking and error handling in here to meet this guarantee.

Reply:
I've added this. On thinking about ID derivation in LocalSelector I also realised that using the file name as the element id is not unique when the directory contains files with the same name but different extensions, so I changed it to use the name and extension concatenated which should guarantee uniqueness. 

Since we now require the each selector's `index` method to return a dataframe with an element_id, checking for uniqueness could be performed in the `start_indexing` of the selector ABC.